iOS/Airbnb_iOS/Airbnb_iOS/Utility/Extension/Int+.swift

```diff
@@ -14,4 +14,18 @@ extension Int {
         let price = numberFormatter.string(from: self as NSNumber) ?? ""
         return "\(price)원"
     }
+
+    func toKRWWithCurrencyMark() -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
```

Comment:
제가 알려주는 것 이외에도 NumberFormatter에서 어떤 것들이 있는지 한 번 검색해보시면 좋을거 같아요🙂

```suggestion
        numberFormatter.numberStyle = .currency
        numberFormatter.locale = Locale(identifier: "ko_KR")
```